.github/workflows/sycl_linux_build_and_test.yml

```diff
@@ -44,10 +44,6 @@ on:
       build_artifact_suffix:
         type: string
         required: true
-      build_upload_artifact:
-        type: boolean
-        required: false
-        default: false
 
```

Comment:
Why this option is removed? Do we want **always** upload artifacts?

Reply:
> Why this option is removed? Do we want **always** upload artifacts?

> The idea is to generate a packaged artifact, which is then distributed to multiple systems to parallelize tests.

So, the answer to my second question is "yes". Right?

Reply:
Right